### PR TITLE
Update Sapio project description and summary

### PR DESCRIPTION
**Pull request by lucasmogu**
Update "summary" and "description" as per OpenSats/volunteers#1

### CODE REVIEW

File: data/projects/sapio.mdx [hunk 1]
```diff
@@ -17,12 +17,10 @@ tags: ['Bitcoin']
 
 Sapio helps you build payment protocol specifiers that oblivious third parties can participate in being none the wiser.
 
-For example, with Sapio you can generate an address that represents a lightning channel between you and friend and give that address to a third party service like an exchange and have them create the channel without requiring any signature interaction from you or your friend, zero trusted parties, and an inability to differentiate your address from any other.
-That’s the tip of the iceberg of what Sapio lets you accomplish.
+For example, with Sapio you can generate an address that represents a lightning channel between you and a friend and give that address to a third-party service like an exchange and have them create the channel without requiring any signature interaction from you or your friend, zero trusted parties, and an inability to differentiate your address from any other. That’s the tip of the iceberg of what Sapio lets you accomplish.
```
Comment (Arvin21M) [inline data/projects/sapio.mdx:20]:
```suggestion
For example, with Sapio you can generate an address that represents a Lightning channel between you and a friend, then give that address to a third-party service like an exchange to have them create the channel without requiring any signature interaction from you or your friend, using zero trusted parties, and an inability to differentiate your address from any other. That’s the tip of the iceberg of what Sapio lets you accomplish.
```

File: data/projects/sapio.mdx [hunk 2]
```diff
@@ -17,12 +17,10 @@ tags: ['Bitcoin']
...
 
-Before Sapio, most Bitcoin smart contracts primarily focused on who can redeem coins when and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc). A few languages, such as BitML, placed emphasis on multi-transaction and multi-party use cases.
-Sapio in particular focuses on transactions using BIP-119 OP_CHECKTEMPLATEVERIFY. OP_CHECKTEMPLATEVERIFY enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
+Before Sapio, most Bitcoin smart contracts primarily focused on who could redeem coins when and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc.). A few languages, such as BitML, emphasized multi-transaction and multi-party use cases. Sapio in particular focuses on transactions using BIP-119 OP_CHECKTEMPLATEVERIFY. OP_CHECKTEMPLATEVERIFY enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
```
Comment (Arvin21M) [inline data/projects/sapio.mdx:22]:
```suggestion
Before Sapio, most Bitcoin smart contracts primarily focused on who could redeem coins when, and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc.). A few languages, such as BitML, emphasized multi-transaction and multi-party use cases. Sapio in particular focuses on transactions using BIP-119 OP_CHECKTEMPLATEVERIFY. OP_CHECKTEMPLATEVERIFY enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
```

File: data/projects/sapio.mdx [hunk 3]
```diff
@@ -17,12 +17,10 @@ tags: ['Bitcoin']
...
 
-Sapio is a tool for defining such smart contracts in an easy way and exporting easy to integrate APIs for managing open contracts. With Sapio you can turn what previously would require months or years of careful tinkering with Bitcoin internals into a 20 minute project and get a fully functional Bitcoin application.
+Sapio is a tool for easily defining such smart contracts and exporting easy-to-integrate APIs for managing open contracts. With Sapio you can turn what previously would require months or years of careful tinkering with Bitcoin internals into a 20-minute project and get a fully functional Bitcoin application.
```
Comment (Arvin21M) [inline data/projects/sapio.mdx:24]:
```suggestion
Sapio is a tool for easily defining such smart contracts and exporting easy-to-integrate APIs for managing open contracts. With Sapio you can turn what previously would require months or years of careful tinkering with Bitcoin internals into a 20-minute project to get a fully functional Bitcoin application.
```

**Review by Arvin21M — CHANGES_REQUESTED** [review]

File: data/projects/sapio.mdx [hunk 4]
```diff
@@ -17,12 +17,10 @@ tags: ['Bitcoin']
...
 
-Before Sapio, most Bitcoin smart contracts primarily focused on who can redeem coins when and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc). A few languages, such as BitML, placed emphasis on multi-transaction and multi-party use cases.
-Sapio in particular focuses on transactions using BIP-119 OP_CHECKTEMPLATEVERIFY. OP_CHECKTEMPLATEVERIFY enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
+Before Sapio, most Bitcoin smart contracts primarily focused on who could redeem coins when, and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc.). A few languages, such as BitML, emphasized multi-transaction and multi-party use cases. Sapio in particular focuses on transactions using BIP-119 OP_CHECKTEMPLATEVERIFY. OP_CHECKTEMPLATEVERIFY enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
```
Comment (dergigi) [inline data/projects/sapio.mdx:22]:
```suggestion
Before Sapio, most Bitcoin smart contracts primarily focused on who could redeem coins when, and what unlocking conditions were required (see Ivy, Policy/Miniscript, etc.). A few languages, such as BitML, emphasized multi-transaction and multi-party use cases. Sapio in particular focuses on transactions using [BIP-119](https://utxos.org/) `OP_CHECKTEMPLATEVERIFY`. `OP_CHECKTEMPLATEVERIFY` enables Bitcoin Script to support complex multi-step smart contracts without a trusted setup.
```

**Review by dergigi — CHANGES_REQUESTED** [review]
I've added some markdown formatting for `OP_CHECKTEMPLATEVERIFY` and a link to [BIP-119](https://utxos.org/). 

You should be able to accept my suggested change with one click.

**Review by Arvin21M — APPROVED** [review]

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Pull request merged**